arch/inst/C/c.slli.yaml

```diff
@@ -11,16 +11,24 @@ definedBy:
   anyOf:
     - C
     - Zca
-base: 64
 assembly: xd, shamt
 encoding:
-  match: 000-----------10
-  variables:
-    - name: shamt
-      location: 12|6-2
-    - name: rd
-      location: 11-7
-      not: 0
+  RV32:
+    match: 0000----------10
+    variables:
+      - name: shamt
+        location: 6-2
```

Comment:
Should have:
```
  not: 0
```
Per the spec, "For RV32C and RV64C, the shift amount must be non-zero".

Same for the RV64 stanza, below.